doc/index.rst

```diff
@@ -6,14 +6,55 @@
 Welcome to petab-select's documentation!
 ========================================
 
+PEtab Select brings
+`model selection <https://en.wikipedia.org/wiki/Model_selection>`_ to
+`PEtab <https://petab.readthedocs.io/>`_. PEtab Select comprises file
+formats, a Python library and a command line interface.
+
+Model selection is the process of choosing the best model from a set of
+candidate models. PEtab Select provides a standardized and compact way to
+specify the candidate model space, implements a number model selection
+algorithms and criteria.
+
+Supported model selection algorithms:
+
+* brute force
+* forward selection
+* backward selection
+* FAMoS
+
+Supported model selection criteria:
+
+* (corrected) Akaike Information Criterion (AIC / cAIC)
+* Bayesian Information Criterion (BIC)
+* ...
+
+Model calibration is performed outside of PEtab Select. For example,
+PEtab Select is well-integrated with
+`pypesto <https://pypesto.readthedocs.io/>`_
+(`model selection example <https://pypesto.readthedocs.io/en/latest/example/model_selection.html>`_).
```

Comment:
These tools all support the current release of PEtab Select. They will need to update some things to support the next release.

```suggestion

* `BasiCO <https://basico.readthedocs.io/>`_ (`example <https://basico.readthedocs.io/en/latest/notebooks/Working_with_PEtab.html#Model-Selection>`_)
* `PEtab.jl <https://sebapersson.github.io/PEtab.jl>`_ (`example <https://sebapersson.github.io/PEtab.jl/stable/pest_select/>`_)
* `pyPESTO <https://pypesto.readthedocs.io/>`_
(`example <https://pypesto.readthedocs.io/en/latest/example/model_selection.html>`_)

```

Reply:
Thanks!